CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 1.1.4
 * Bugfix to address field value to column name mapping for Tuples
+* Adding a KeyToValue transformation to allow for key to be stored in a separate column in ClickHouse
```

Comment:
@Paultagoras  it's new functionality, thus the version must be `1.2.0` to follow SemVer